gamestonk_terminal/cryptocurrency/discovery/dappradar_view.py

```diff
@@ -0,0 +1,182 @@
+"""DappRadar view"""
+__docformat__ = "numpy"
+
+import os
+from gamestonk_terminal.cryptocurrency.dataframe_helpers import (
+    very_long_number_formatter,
+)
+from gamestonk_terminal.cryptocurrency.discovery import dappradar_model
+from gamestonk_terminal.helper_funcs import (
+    export_data,
+    rich_table_from_df,
+)
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.rich_config import console
+
+
+def display_top_nfts(top: int = 10, sortby: str = "", export: str = "") -> None:
+    """Displays top nft collections [Source: https://dappradar.com/]
+
+    Parameters
+    ----------
+    top: int
+        Number of records to display
+    sortby: str
+        Key by which to sort data
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df = dappradar_model.get_top_nfts()
+    if df.empty:
+        console.print("Failed to fetch data from DappRadar\n")
+    else:
+        if sortby in dappradar_model.NFT_COLUMNS:
+            df = df.sort_values(by=sortby, ascending=False)
+        for col in ["Floor Price [$]", "Avg Price [$]", "Market Cap [$]", "Volume [$]"]:
+            if col in df.columns:
+                df[col] = df[col].apply(lambda x: very_long_number_formatter(x))
+        if gtff.USE_TABULATE_DF:
+            rich_table_from_df(
+                df.head(top),
+                headers=list(df.columns),
+                floatfmt=".2f",
+                show_index=False,
+                title="Top NFT collections",
+            )
+            console.print("")
+        else:
+            console.print(df.to_string, "\n")
+
+    export_data(
```

Comment:
We don't want to export in case the data frame is empty.

I think instead of doing `if else` to have less indentation what I usually do is a return, that makes it so that from then on you know that you have the data needed to work on.